normalizer_test.go

```diff
@@ -899,6 +939,38 @@ func TestNormalizerStoredProcedure(t *testing.T) {
 	}
 }
 
+func TestNormalizerWithoutSpaceBetweenParentheses(t *testing.T) {
```

Comment:
I think this looks good but we might want to refactor the tests eventually to have test cases with `input`, `expected` _and_ a set of options. That way, tests can be collocated for the same input but with different options and really clearly show the difference in expected result depending on those options.

Reply:
yeah that is on my list.